### PR TITLE
1670/feat/search validated contact

### PR DESCRIPTION
**Pull request by y-golde**
see ticket

### CODE REVIEW

File: client/src/components/App/Content/InvestigationForm/TabManagement/ExposuresAndFlights/Forms/ExposureForm/ExposureForm.tsx [hunk 1]
```diff
@@ -95,17 +141,13 @@ const ExposureForm = (props: Props) => {
...
 										value={exposureSourceSearchString}
 										test-id='exposureSource'
-										onSearchClick={setOptionalCovidPatientsAsync}
-										onKeyDown={(e: React.KeyboardEvent) => {
-											if (e.key === 'Enter') {
-												e.preventDefault();
-												setOptionalCovidPatientsAsync()
-											}
-										}}
+										onSearchClick={() => {}}
```
Comment (chenFriedman) [inline client/src/components/App/Content/InvestigationForm/TabManagement/ExposuresAndFlights/Forms/ExposureForm/ExposureForm.tsx:144]: maybe you can remove: onKeyDown and onSearchClick

File: client/src/models/ExposureForm/PersonalDetailsQueryParams.ts [hunk 2]
```diff
@@ -0,0 +1,8 @@
+
```
Comment (chenFriedman) [inline client/src/models/ExposureForm/PersonalDetailsQueryParams.ts:1]: space

File: client/src/models/ExposureForm/PersonalDetailsQueryParams.ts [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+interface PersonalDetailsQueryParams {
+    name: string;
+    phoneNumber: string;
+}
+
```
Comment (chenFriedman) [inline client/src/models/ExposureForm/PersonalDetailsQueryParams.ts:6]: can remove also this space

Comment (y-golde) [reply client/src/models/ExposureForm/PersonalDetailsQueryParams.ts]: fixed

File: client/src/models/ExposureForm/PersonalDetailsQueryParams.ts (hunk 2 above)
Comment (y-golde) [reply client/src/models/ExposureForm/PersonalDetailsQueryParams.ts]: fixed

File: client/src/components/App/Content/InvestigationForm/TabManagement/ExposuresAndFlights/Forms/ExposureForm/ExposureForm.tsx (hunk 1 above)
Comment (y-golde) [reply client/src/components/App/Content/InvestigationForm/TabManagement/ExposuresAndFlights/Forms/ExposureForm/ExposureForm.tsx]: fixed

**Review by MagusM — APPROVED** [review]

**Review by chenFriedman — APPROVED** [review]

**Pull request merged**